frontend/src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import sidebarStyles from './sidebar.module.scss';
+
+const Sidebar = () => {
+  const buttonList = [
+    { name: 'Summary' },
+    { name: 'Rota' },
+    { name: 'Employees' },
+  ];
```

Comment:
It might make sense to pull the hardcoded stuff out of the function and possibly keep it with other constants. That way if we need to change the sidebar we won't have to modify the body of the function which is a small win.